string name
int32 id
---
bool success
